### ISSUE TITLE
Faced exception after build and then app crashed

### ISSUE THREAD

**Issue by Ali-Alizadeh** [pichillilorenzo/flutter_inappwebview#304]
## Environment

**Flutter version:** 1.12.13+hotfix.8
**Plugin version:**  2.1.0+1
**Android version:** 8.1.0
**Device information:**  Huawei Y7 Prime 2019 (DUB-LX1)

## Description

Added the plugin to an empty project but then the app crashed after build.

**Expected behavior:** 

**Current behavior:** 

## Steps to reproduce

1. flutter create --androidx -a java app_name
2. Add `flutter_inappwebview: ^2.1.0+1` to pubspec.
3. Run the project

## Stacktrace/Logcat   
```lt build\app\outputs\apk\debug\app-debug.apk.
D/FlutterActivity( 4989): Using the launch theme as normal theme.
D/FlutterActivityAndFragmentDelegate( 4989): Setting up FlutterEngine.
D/FlutterActivityAndFragmentDelegate( 4989): No preferred FlutterEngine was provided. Creating a new FlutterEngine for this FlutterFragment.
W/FlutterEngine( 4989): Tried to automatically register plugins with FlutterEngine (io.flutter.embedding.engine.FlutterEngine@20b2966) but could not find and invoke the GeneratedPluginRegistrant.
D/FlutterActivityAndFragmentDelegate( 4989): Attaching FlutterEngine to the Activity that owns this Fragment.
Exited (sigint)```

**Comment by zhangyc** [comment]
你解决了吗？

**Comment by pichillilorenzo** [comment]
Released new version `3.0.0`!
